src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -69811,7 +70014,7 @@ public void Constraints_146()
 @"#nullable disable
 partial class A
 {
-#pragma warning enable nullable
```

Comment:
>#pragma warning enable [](start = 0, length = 22)

@alekseyts @gafter Now that we've removed support for `#pragma warning ... nullable`, I wonder if we want to keep `#pragma warning *enable*` around. This seems like an orthogonal feature to nullability analysis and we no longer need it to support nullability scenarios.

Reply:
If we decide to keep the *enable* setting in `#pragma`, then we need to put some test coverage back. All the tests in this file were converted to `#nullable ...`

---
In reply to: [294563233](https://github.com/dotnet/roslyn/pull/36464#discussion_r294563233) [](ancestors = 294563233)

Reply:
Would like to know if there's a consensus on whether `#pragma warning <setting> nullable` is going to stick around. It does seem a bit redundant in the face of `#nullable <setting> warnings`, but it also feels a bit inconsistent with the fact that `/NoWarn:nullable` can be provided on the command line.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/36550](https://github.com/dotnet/roslyn/issues/36550) to track the question of the `enable` setting (should we keep it or revert it before shipping C# 8?).

---
In reply to: [295000903](https://github.com/dotnet/roslyn/pull/36464#discussion_r295000903) [](ancestors = 295000903)